.eslintrc.json

```diff
@@ -0,0 +1,30 @@
+{
+  "env": {
+    "browser": true,
+    "jasmine": true,
+    "jquery": true,
+    "amd": true
+  },
+  "extends": "airbnb-base/legacy",
```

Comment:
For reviewers: this comes from here:

https://www.npmjs.com/package/eslint-config-airbnb-base